### DISCUSSION TITLE
Doc clarification: extra_files simply adds files to the GitHub repo, not to the executable location?

### DISCUSSION THREAD

**Discussion by tomcam** [goreleaser/goreleaser#1615] [Ideas, Issue Triage, and General discussions]
I would like to ship a subdirectory with additional assets (toml files, css files, etc.) with my app. I thought extra_files would work for that. AFAIK it simply ensures they appear in the public repo but are not copied to my executable's directory, correct?

**Comment by caarlos0** [comment]
`release.extra_files` adds files to the release.

if you want to add files to your archives, check [`archives.files` on docs](https://goreleaser.com/customization/archive/)

**Reply by tomcam** [reply]
Thanks much! Appreciate goreleaser to no end!
